tools/rector.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
not sure about adding this folder

Reply:
if we want to do it, it should be discussed in a separate PR

Reply:
My idea was, if its working, you can upmerge, execute rector and remove this folder afterwards

Reply:
let's remove it from the PR IMHO